vllm/model_executor/layers/quantization/utils/quant_utils.py

```diff
@@ -405,6 +408,8 @@ def unpack_cols(
 
     orig_device = packed_q_w.device
 
+    packed_q_w = packed_q_w.t()
```

Comment:
Is this function only used for machete?

Reply:
Good catch, that was cruft, machete uses `unpack_quantized_values_into_int32` now